CHANGELOG.md

```diff
@@ -7,25 +7,32 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 None
 
-a name="2.0.3"></a>
+<a name="2.1.0"></a>
+## [2.1.0] - 2021-11-02
+
+### Added
+- Added Typescript Support
+- Added `active` optional filter to get call list
```

Comment:
I think we need to update the typescript typings for this new setting/option

Reply:
This should be taken care of now